src/main/resources/routes

```diff
@@ -33,6 +35,9 @@ GET         /application/:id/section/:num/item/:itemNum/edit          controller
 GET         /reset                                                    controllers.ApplicationController.reset
 GET         /wip                                                      controllers.OpportunityController.wip(backUrl: String)
 
+# Manage Views
+GET         /manage/opportunities/guidance/:opportunityId           controllers.OpportunityController.showPMGuidancePage(opportunityId:OpportunityId)
```

Comment:
I don't think it makes sense to have the `opportunityId` as part of the url path. Isn't the text of the guidance page the same for all opportunities?

I know that the guidance template needs to generate a link back to the opportunity detail, but we could pass it in as a query parameter rather than a path segment. If you change this to `/manage/opportunities/guidance` then the play parameter to the controller as a query parameter.